rfcs/endpoints-needed.md

```diff
@@ -0,0 +1,50 @@
+# Enpoints Needed For Each Page
+
+Note all enpoints describe what the endpoint should do not necassarily the name of the endpoint.
+
+### Sign In Page
+
+- Get Username
+- Get Password
+- Check for match (username and password)
+- Directs to Match List Page
+
+  #### Sign Up Page  
+
+  - Check for valid creditials (unique, formatting)
+
+  #### Forgot Password Page
+
+  - Get Username 
+  - Send email with tempory creditials
+
+### Match List Page
+
+- Get all matches (filter by signed in account)
+- Create Match button (sends to create match page)
+- Match Details button (sends to match details page)
+- Get user information (logo, username, team name)
+- Get NFL week 
+
+### Match Details Page
+
+- Add Player Button (Player List Page)
+- Get user information (logo, username, team name)
```

Comment:
I think we can pass in this information as a state from the Match List page. That way, we don't need another API request (I think we did something similar to this when clicking onto the details page for the Pokemon cards)